templates/docker-compose.yml.j2

```
@@ -42,18 +42,13 @@ services:
 {% for protocol in nim_waku_protocols_enabled %}
       --{{ protocol }}=true
 {% endfor %}
-      --db-path=/data
       --rpc-admin=true
-      --persist-peers=true
+      --peer-persistence=true
       --keep-alive=true
-      --persist-messages=true
       --max-connections={{ nim_waku_p2p_max_connections }}
-      --store-capacity={{ nim_waku_store_capacity }}
       --dns4-domain-name={{ nim_waku_dns4_domain_name }}
-{% if nim_waku_sqlite_store %}
-      --sqlite-store=true
-      --sqlite-retention-time={{ nim_waku_sqlite_retention_time }}
-{% endif %}
+      --store-message-db-url='sqlite:///data/store.sqlite3'
```

We should probably make this conditional on store being used:
```suggestion
{% if "store" in nim_waku_protocols_enabled %}
      --store-message-db-url='sqlite:///data/store.sqlite3'
```
With a closing `endif` of course.

---

Done in [8fccbc8](https://github.com/status-im/infra-role-nim-waku/pull/6/commits/8fccbc8e17c46121617a5838be73a133935c8c84)